examples/aliens.py

```diff
@@ -186,7 +187,7 @@ class Bomb(pg.sprite.Sprite):
     """A bomb the aliens drop."""
 
     speed = 9
-    images = []
+    images: List = []
```

Comment:
Would be nice to have the type hinting narrowed down further, wherever a plain `List` is used... it should be `List[type]`. 

PS: `Union`s can be used if the lists are heterogenous but you probably know this already

Reply:
What data types does this list hold?

Reply:
```suggestion
    images: List[pg.Surface] = []
```